core/src/test/java/org/wso2/carbon/kernel/internal/transports/DummyInterpreter.java

```diff
@@ -0,0 +1,75 @@
+package org.wso2.carbon.kernel.internal.transports;
+
+import org.eclipse.osgi.framework.console.CommandInterpreter;
+import org.osgi.framework.Bundle;
+
+import java.util.Dictionary;
+
+public class DummyInterpreter implements CommandInterpreter {
+    private String[] transportIdList;
+    private int counter;
+
+    public void setTransportIdList(String[] list) {
+        transportIdList = list;
+        counter = 0;
+    }
+
+    public void resetCounter() {
+        counter = 0;
+    }
+
+    public void setTransportIdListValuesToNull() {
```

Comment:
Would be better to add javadocs and explain what's being done through the method